test/src/test/java/org/corfudb/runtime/clients/NettyCommTest.java

```diff
@@ -682,6 +697,7 @@ void bootstrapServer() {
                             nsr,
                             address,
                             Integer.parseInt((String) serverContext.getServerConfig().get("<port>")));
+            f.syncUninterruptibly();
```

Comment:
Can you please comment on what line is responsible for delaying bootstrap server?

Reply:
The test was failing at `pingSync()`
> Expecting value to be true but was false

Technically the ciphers and trust stores should work all of the time or none of the time.
Hence this error indicates the server was not available at the time `pingSync()` was called.

`f.syncUninterruptibly();` is called [here](https://github.com/CorfuDB/CorfuDB/blob/9f8422eb09bf386bddd790bad49d46e28f1b0a80/infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServerNode.java#LL177C7-L177C7), but was missing from these test classes. Hence, I added it.

Moreover, I think the failure was also in clean up where we were shutting down the threads after each test.
I moved all the helper method calls into individual tests to ensure the cleanup.